src/Parser/WindwalkerMonk/Modules/Spells/SEF.js

```diff
@@ -0,0 +1,40 @@
+import Module from 'Parser/Core/Module';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import SPELLS from 'common/SPELLS';
+
+class SEF extends Module{
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  sefCDReduction = 0;
```

Comment:
This value is only set if `reducedCooldownWithTraits` is used before it. If it's not used it is 0. This would be error-prone. I think you should be able to turn this into a getter by moving lines 13-34 here.

Reply:
You want me to move sefCDReduction into the get reducedCooldownWithTraits() ?
I followed the way the Trueshot class was set up for marksmanship, so had imagined the layout wouldn't be problematic.

Reply:
No I meant something else. The value of `sefCDReduction` is set within `reducedCooldownWithTraits`, but I think making `sefCDReduction` a getter like `reducedCooldownWithTraits` that returns just its own value you can have both work without interfering with each other.

Reply:
I believe latest commit should be what you're asking for

Reply:
I think this part:
https://github.com/Juko8/WoWAnalyzer/blob/3d809b881f17913a04e330029a9deb8d3ba42320/src/Parser/WindwalkerMonk/Modules/Spells/StormEarthAndFire.js#L16-L37
should be the contents of  sefCDReduction`.

Also I think a better name for `sefCDReduction` would be `traitsCooldownReduction`. It's in the SEF class so prefixing it with that is unnecessary (you wouldn't do `car.carSpeed`).

Reply:
Alright, i believe i've gotten it looking like that now